### ISSUE TITLE
Extra root in reported errors' file path since 4.0.5

### ISSUE THREAD

**Issue by guo-sun** [ionide/ionide-vscode-fsharp#1177]
**Describe the bug**
Extra `C:\` appears in file path of reported problems since 4.0.5

**To Reproduce**
Steps to reproduce the behaviour:
1. Open F# file with an error

**Expected behaviour**
Errors are reported with just one `C:\` in the file path

**Screenshots**
4.0.4 works
![4 0 4-works](https://user-images.githubusercontent.com/6422188/61085226-5632de00-a3f5-11e9-9ee8-42a351024193.PNG)

4.0.5 with extra root prefix
![4 0 5](https://user-images.githubusercontent.com/6422188/61085228-5632de00-a3f5-11e9-89fa-49737022154f.PNG)

**Environment (please complete the following information):**
 - OS: Windows 10 64bit 1803
 - Ionide version: 4.0.5
 - VSCode version: 1.36.1

**Comment by ssmock** [comment]
I'm seeing similar behavior in VS Code's "Problems" pane even for projects without any problems. (I haven't experienced problems on Ubuntu with the same project.)

![image](https://user-images.githubusercontent.com/4741921/61089342-ceef6580-a408-11e9-99c9-782d7709eac7.png)

* OS: Windows 10 Pro, 1803
* Ionide 4.0.5
* VS Code 1.36.1

**Comment by baronfel** [comment]
Sample from a windows machine opening Saturn:

```
[Trace - 10:25:23 PM] Received notification 'textDocument/publishDiagnostics'.
Params: {
    "uri": "file:///d:/d:/code/Saturn/src/Saturn/Common.fs",
    "diagnostics": [
        {
            "range": {
                "start": {
                    "line": 3,
                    "character": 15
                },
                "end": {
                    "line": 3,
                    "character": 25
                }
            },
            "severity": 1,
            "code": "39",
            "source": "F# Compiler",
            "message": "The namespace 'AspNetCore' is not defined.",
            "relatedInformation": []
        }
    ]
}
```
it seems like I introduced a windows-specific bug here in the changes to `filePathToUri`, but I'd have to do more digging to figure out what the actual input to `filePathToUri` was here to trigger this behavior.

**Comment by airmanx86** [comment]
Confirming that this is a 4.0.5 problem on Windows 10. Half of my team suffer from this problem since about 30 mins ago and going back to 4.0.4 resolve the issue.

**Comment by baronfel** [comment]
Yes that would be a good workaround. I'm working on finding the 'root'-cause of the issue on my windows machine now, then it will require an update to FSAutoComplete and a new version of Ionide.  Again, my apologies for the regression.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `4.0.6`
